tasks/discover-pdrs/README.md

```diff
@@ -4,20 +4,6 @@ Discover PDRs in FTP/HTTP/HTTPS/SFTP/S3 endpoints
 
 ## Message Configuration
 
-### Config
```

Comment:
Removed this for consistency/duplication reasons, matches what's in discoverGranules -- this information is conveyed in the schema.json and referenced here https://nasa.github.io/cumulus/docs/tasks